data/timesketch.conf

```diff
@@ -180,7 +180,6 @@ NEO4J_PASSWORD = '<NEO4J_PASSWORD>'
 # Analyzers.
 
 # Which analyzers to run automatically.
-AUTO_INDEX_ANALYZERS = []
```

Comment:
Is there a way to tell users who have an older version of the config that the key has been deprecated?

Reply:
not sure we need to, since there haven't been any INDEX analyzers to pick ... well ever, so this is should be a no-op

Reply:
SGTM